addons/website_sale/views/templates.xml

```diff
@@ -936,7 +936,7 @@
                             <t t-call="website_sale.shop_product_images"/>
                         </div>
                         <div t-attf-class="col-lg-#{image_cols[1]} mt-md-4" id="product_details">
-                            <t t-set="base_url" t-value="product.get_base_url()"/>
+                            <t t-set="base_url" t-value="website.get_base_url()"/>
```

Comment:
@robinlej As a stable fix in XML, always *add* instead of change/remove if possible. In this case, it's very possible: just add a second t-set.
```xml
<t t-set="base_url" t-value="website.get_base_url()"/>
<t t-if="False" t-set="base_url" t-value="product.get_base_url()"/> <!-- add a comment -->
```
Then, in master, we'll remove the useless one.

Advantage: if someone has a custo that targets `t-value="product.get_base_url()"` (potentially to do the same fix that you are doing here), you are not breaking their database on update. I am also adding the new correct one before the previous one (cancelling the previous one with a `t-if="False"`, so that if someone has a custo that target `t-set="base_url"` (most probable custo on this), they are still changing it the same way. That's just a bonus though, you can still find custo that would be broken after my suggestion. The important thing is to not break the update IMO (and of course, avoid changing any XML if not required).

Reply:
I think about it when it targets usual tags/attributes, but it didn't cross my mind for the t-value. That `t-if="False"` trick is nice in that situation, thanks :+1: